README.md

```diff
@@ -7,10 +7,23 @@ CWQ is designed for fast searching and presentation of log data in multiple form
 The basic command structure is:
 
 ```bash
-cwq --log-group <log-group> '<query>'
```

Comment:
I would recommend we keep this as it was, since it sits right at the top of the README and describes the typical usage. I imagine it will be rare that users are using this for a way to pull logs without a query.

Reply:
Only use lines 15 - 20: 06a25cf7cb31bb0c8129ee1e7ac3b4fde27f3895.